### PR TITLE
Set Ownership of the archive dir

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

Adds two new variables `archive_owner` and `archive_group`

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Updated a test scenario

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: VARIABLES.md [hunk 1]
```diff
@@ -196,6 +196,22 @@ Default:  "/opt/confluent"
 
 ***
 
+### archive_owner
```
Comment (domenicbove) [inline VARIABLES.md:199]: not sure about these var names

**Comment by JumaX** [comment]
@domenicbove why is this required?  The archive will downloaded and owned by the user logged into the system, as with any downloaded file.  This is expected behaviour, so I am confused why we would need to change this, as generally you would unarchive to the desired location as the user who is expected to run the contents.

**Comment by domenicbove** [comment]
@JumaX With this PR, the default behavior stays the same- ie no ownership gets set, thats what the `omit` does. Some customers requested that they could set ownership on the expanded dir, those customers would have to use the new variables to do so. They did not want the ansible user to own the directory.

**Review by JumaX — APPROVED** [review]
LGTM.

**Pull request merged**